experiments/superdarn_common_fields.py

```diff
@@ -6,6 +6,9 @@
 
 from utils.experiment_options.experimentoptions import ExperimentOptions
 
+# TODO: We should protect these values from changing, I noticed during testing that I used a
+# TODO: call to reverse() on one and it affected the rest of the testing afterwards
```

Comment:
good point - maybe these need to be class properties like the config variables.

Reply:
I'll make an issue for a future PR